core/src/conn.rs

```diff
@@ -22,14 +22,23 @@ impl MongoConnection {
     // The operation will timeout if it takes more than loginTimeout seconds.
     // The initial current database if provided should come from SQL_ATTR_CURRENT_CATALOG
```

Comment:
@nbagnard any idea where we should read the value of `SQL_ATTR_CURRENT_CATALOG`?

Reply:
Not sure where I got the idea to pull `SQL_ATTR_CURRENT_CATALOG` from the shell environment!